packages/contracts-bedrock/test/governance/MintManager.t.sol

```diff
@@ -23,10 +24,20 @@ contract MintManager_Initializer is CommonTest {
         super.setUp();
 
         vm.prank(owner);
-        gov = IGovernanceToken(address(new GovernanceToken()));
+        gov = IGovernanceToken(
+            DeployUtils.create1({
```

Comment:
Should be able to get rid of the imports for `GovernanceToken` and `MintManager`